src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,87 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final User nullUser = null;
+    private static final User nullLoginUser = new User(null, "1234567", 11);
+    private static final User nullPasswordUser = new User("????", null, 16);
+    private static final User nullAgeUser = new User("????", "????????", null);
+    private static final User illegalAgeUser = new User("Pudj2008", "asdfg2002", 13);
+    private static final User illegalAgeSecondUser = new User("Phgedsa8", "asdhdr", -124);
+    private static final User illegalPasswordLengthUser = new User("Pudj2002", "asd", 18);
+    private static final User Bohdan = new User("Bohdan", "password", 58);
+    private static final User Dan = new User("Dan", "NothingPassword", 68);
+    private static final User Igor = new User("ar1n04cka", "ytrewq1236", 27);
+    private static final User anotherIgor = new User("Igor1992", "Igor123456", 24);
```

Comment:
Usually, it is bad practice to make objects constant, you should make constant some value that you use for user initialization. Also, you can create a default user and set some fields in a specific test method.